lua/entities/prop_trash.lua

```diff
@@ -340,5 +343,13 @@ function ENT:CanTape(userid)
 end
 
 function ENT:OnShoot()
-	self:UnTape()
+	if Safe(self) then
+		self.ProtectedPropShot = self.ProtectedPropShot + 1
+	end
+
+	if self.ProtectedPropShot == 6 and Safe(self) then
+		self:UnTape()
```

Comment:
?

Reply:
fixed

Reply:
`if self.ProtectedPropShot == 6 or not Safe(self) then`